Include the custom policies note about preview

fmigacz-zz opened the pull request:
The Guides pages and the FAQ pages include statements about custom policies being in preview. I added a link to that Note in this article.

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Commit:
Include the custom policies note about preview

The guides pages and the FAQ pages include statements about custom policies being in preview. I added a link to that Note in this article.
